src/components/InputMoney.js

```diff
@@ -36,85 +38,105 @@ class InputMoney extends Component {
     maxLength: PropTypes.number,
     balance: PropTypes.string,
     getRef: PropTypes.func,
-    onChange: PropTypes.func
+    onChange: PropTypes.func,
+    pennies: PropTypes.bool
   };
 
   static defaultProps = {
     prefix: "£",
     maxLength: 10,
     balance: "",
     getRef: null,
-    onChange: null
+    onChange: null,
+    pennies: true
   };
 
   state = {
-    remaining: ""
+    remaining: "",
+    acitve: false,
+    focus: false
   };
 
   componentDidMount() {
     if (typeof this.props.getRef === "function") this.props.getRef(this.input);
   }
 
   _onChange = value => {
-    const formatedValue = formatAmount(value);
+    let formatedValue;
+    if (typeof value === "string") {
+      formatedValue = !this.props.pennies
```

Comment:
instead of `!this.props.pennies` which is just GBP also, we could invert to `this.props.noDecimals`